From eddd5f6061f8d51ff4ac1113ce677b5d7cb80b13 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Mon, 5 Dec 2016 23:12:05 +0800
Subject: [PATCH 1307/5242] MLK-13566 ARM: imx: update LPDDR3 script on
 imx7ulp som board

commit  6d490d5e7c7ab28d68c639957b84cbba73585614 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update i.MX7ULP SOM board LPDDR3 script according to
u-boot script change.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/mach-imx/pm-imx7ulp.c |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm/mach-imx/pm-imx7ulp.c b/arch/arm/mach-imx/pm-imx7ulp.c
index fea87df..612e193 100644
--- a/arch/arm/mach-imx/pm-imx7ulp.c
+++ b/arch/arm/mach-imx/pm-imx7ulp.c
@@ -193,11 +193,11 @@ struct imx7ulp_pm_socdata {
 	0x834, 0x838, 0x8c0, 0x8b8,
 	0x004, 0x00c, 0x010, 0x038,
 	0x014, 0x018, 0x02c, 0x030,
-	0x040, 0x000, 0x83c, 0x01c,
+	0x040, 0x000, 0x01c, 0x01c,
 	0x01c, 0x01c, 0x01c, 0x01c,
 	0x01c, 0x01c, 0x01c, 0x01c,
-	0x01c, 0x020, 0x800, 0x004,
-	0x404, 0x01c,
+	0x01c, 0x01c, 0x83c, 0x020,
+	0x800, 0x004, 0x404, 0x01c,
 };
 
 static const u32 imx7ulp_lpddr3_script[] __initconst = {
@@ -207,11 +207,11 @@ struct imx7ulp_pm_socdata {
 	0xf3333333, 0xf3333333, 0x24922492, 0x00000800,
 	0x00020052, 0x292C42F3, 0x00100A22, 0x00120556,
 	0x00C700DB, 0x00211718, 0x0F9F26D2, 0x009F0E10,
-	0x0000003F, 0xC3190000, 0x20000000, 0x003F8030,
-	0x003F8038, 0xFF0A8030, 0xFF0A8038, 0x04028030,
-	0x04028038, 0x83018030, 0x83018038, 0x01038030,
-	0x01038038, 0x00001800, 0xA1310000, 0x00020052,
-	0x00011006, 0x00000000,
+	0x0000003F, 0xC3190000, 0x00008050, 0x00008058,
+	0x003F8030, 0x003F8038, 0xFF0A8030, 0xFF0A8038,
+	0x04028030, 0x04028038, 0x83018030, 0x83018038,
+	0x01038030, 0x01038038, 0x20000000, 0x00001800,
+	0xA1310000, 0x00020052, 0x00011006, 0x00000000,
 };
 
 static const struct imx7ulp_pm_socdata imx7ulp_lpddr3_pm_data __initconst = {
-- 
1.7.9.5

